src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -2713,6 +2726,13 @@ namespace PurchasedAirManager {
                 Node(InNodeNum).GenContam = Node(ZoneNodeNum).GenContam;
             }
 
+            if (PurchAir(PurchAirNum).EMSOverrideMdotOn) {
+                PurchAir(PurchAirNum).EMSValueMassFlowRate = 0.0;
```

Comment:
this looks odd, only EMS programs should be setting such a value

Reply:
@EnergyArchmage What do you suggest? My intent was to set the value=0, since no cooling and heating.